Missionframework/functions/curator/fn_initCuratorHandlers.sqf

```diff
@@ -0,0 +1,87 @@
+/*
+    File: fn_initCuratorHandlers.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2020-08-07
+    Last Update: 2020-08-07
+    License: MIT License - http://www.opensource.org/licenses/MIT
+
+    Description:
+        Initialize curator handlers.
+
+    Parameter(s):
+        NONE
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+#define ZEUSVAR(var) (format ["KPLIB_zeus_%1", var])
+
+if (isServer) then {
+
+    // delete old Liberation mission placed Zeus module
+    deleteVehicle zm1;
```

Comment:
Maybe concerning derivatives which might have other modules, just delete all current available curator modules in the mission to avoid any false positives?
Or really just the zm1 from our template, so additional modules wouldn't be affected and therefore might not influence derivative plans?

Reply:
IMHO removing zm1 is enough. People might have added additional full zeuses for other slots manually in their edits. We should not touch that.